content/articles/manage-ds-record.md

```diff
@@ -7,33 +7,38 @@ categories:
 ---
 
 # Managing DS Records
-
+ 
 ### Table of Contents {#toc}
 
 * TOC
 {:toc}
 
 ---
 
+DS (Delegation Signer) records are used in [DNSSEC](/articles/dnssec/) to secure your domain.
+
 ## Locating the DS records page
 
-To view and manage the delegation signer (DS) records for a domain, click on the DNSSEC tab on a domain's management page, then click on the <label>Manage</label> link in the DS Records section.
+To view and manage the delegation signer (DS) records for a domain:
 
-![Manage DS Records link](/files/ds-records-manage.png)
+1. In your Domain List, click the domain name to acccess the domain's management page.
+1. Click the **DNSSEC** tab on the left side, then click **Manage** in the **DS Records** card.
 
-## Listing DS records
+<!--- needs screenshot -->
 
-On the DS Records page, you will be able to see a list of the DS records currently provisioned for the domain.
+## Listing DS records
 
-![List DS Records](/files/ds-records-list.png)
+On the DS Records page, you will see a list of the DS records currently provisioned for the domain.
 
 ## Adding a DS record
 
-On the DS records page, click <label>Add DS record</label>.
+<info>
+The form fields shown on the Add DS record page will differ based on whether the TLD of the domain requires them to be set up with the DS-Data interface or the KEY-Data interface. This is automatically populated based on the TLD.
```

Comment:
I found this little nugget buried in the steps. Felt important so I upgraded it

Reply:
Good call